.travis.yml

```diff
@@ -34,6 +43,7 @@ before_script:
 
 script:
   - ./gradlew check :grpc-all:jacocoTestReport
+  - bazel build //compiler:all //context:all //core:all //netty:all //okhttp:all //protobuf:all //protobuf-lite:all //protobuf-nano:all //stub:all //testing:all
```

Comment:
This can be `//...`  right?

Reply:
+1

Reply:
Slightly different and `...` will unintentionally include `//examples:all` which is a non-target.

Reply:
If examples is the only reason for the expansion, then we should subtract it out:

```
bazel build -- ... -examples/...
```

Also `/...` should be preferred to `:all` here, since there's no need to disable descending into subpackages (although they don't exist today).

Honestly, I would be okay with including the examples in this line. I know that users will see something slightly different since the WORKSPACE is handled differently, but building examples using its own WORKSPACE will rebuild everything a _third_ time.

Reply:
I will change building each services to `...`.